hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/consumer/sender/http/HttpClientFactory.java

```diff
@@ -2,29 +2,39 @@
 
 import org.eclipse.jetty.client.HttpClient;
 import org.eclipse.jetty.util.HttpCookieStore;
+import org.eclipse.jetty.util.ssl.SslContextFactory;
 import org.glassfish.hk2.api.Factory;
 import pl.allegro.tech.hermes.common.config.ConfigFactory;
 import pl.allegro.tech.hermes.common.metric.executor.InstrumentedExecutorServiceFactory;
 
 import javax.inject.Inject;
+import java.io.File;
+import java.security.KeyStore;
+import java.util.Arrays;
 import java.util.concurrent.ExecutorService;
 
-import static pl.allegro.tech.hermes.common.config.Configs.*;
+import static java.util.stream.Collectors.joining;
+import static pl.allegro.tech.hermes.common.config.Configs.CONSUMER_HTTP_CLIENT_MAX_CONNECTIONS_PER_DESTINATION;
+import static pl.allegro.tech.hermes.common.config.Configs.CONSUMER_HTTP_CLIENT_THREAD_POOL_MONITORING;
+import static pl.allegro.tech.hermes.common.config.Configs.CONSUMER_HTTP_CLIENT_THREAD_POOL_SIZE;
+import static pl.allegro.tech.hermes.common.config.Configs.CONSUMER_INFLIGHT_SIZE;
 
 public class HttpClientFactory implements Factory<HttpClient> {
 
     private final ConfigFactory configFactory;
     private final InstrumentedExecutorServiceFactory executorFactory;
+    private final SslContextFactory sslContextFactory;
 
     @Inject
     public HttpClientFactory(ConfigFactory configFactory, InstrumentedExecutorServiceFactory executorFactory) {
         this.configFactory = configFactory;
         this.executorFactory = executorFactory;
+        this.sslContextFactory = sslContextFactory();
```

Comment:
right, this could be injected, but anyway it has a very narrow scope as it's used only by the http client.